frontend/src/pages/Home.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import styled from 'styled-components/macro';
+import { Wrapper } from 'lib'
```

Comment:
There is a missing semi colon on this line. the "./" are also missing from this import. And it's good to have a line break here. Between local imports and external.

Reply:
Fixed!